vendordeps/PathfinderOLD.json

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
Is there a reason this is called `PathfinderOLD.json`? Just wondering if it was put in here accidentally.

Reply:
It's correct because there's a new Pathfinder version for 2019 that's still under development.

Reply:
That was the file that the Pathfinder version 1 setup instructions said to install.